packages/react-native-renderer/src/ReactNativeComponentTree.js

```diff
@@ -26,8 +26,9 @@ function getTagFromInstance(inst) {
   let tag = nativeInstance._nativeTag;
```

Comment:
Should we rename these methods? It's already exported as `getNodeFromInstance`

Reply:
I still need to do some more work on the react repo for this. Can do that in a following PR.